implementation/test/DepositUtilsTest.js

```diff
@@ -376,7 +382,61 @@ describe("DepositUtils", async function() {
   })
 
   describe("auctionValue()", async () => {
-    it.skip("is TODO")
+    let duration
+    let basePercentage
+    before(async () => {
+      duration = await tbtcConstants.getAuctionDuration.call()
+      basePercentage = await tbtcConstants.getAuctionBasePercentage.call()
+      auctionValue = new BN(100000000)
+    })
+    beforeEach(async () => {
+      await createSnapshot()
+    })
+
+    afterEach(async () => {
+      await restoreSnapshot()
+    })
+
+    it("returns base value if no time has elapsed", async () => {
+      testDeposit.send(auctionValue, {from: accounts[0]})
+      const block = await web3.eth.getBlock("latest")
+
+      await testDeposit.setLiquidationAndCourtesyInitated(block.timestamp, 0)
+
+      const value = await testDeposit.auctionValue.call()
```

Comment:
Almost feels like we need a contract helper that sets the timestamp and immediately returns the auction value here… What if a block is mined between the `setLiquidationAnd…` call above and this call?

Reply:
I've seen that a grace of +1 second can be used.  think it's okay to test within that margin. ie: value is either as expected, or what would be expected given we're off by 1 second.

Reply:
I'm mostly thinking about what happens if we're running tests in parallel. Realize we're not doing this right now, but I want to get there at some point as a performance optimization.

We can deal with it then, but having a helper as I described above feels like a more correct way to express the test we're trying to express here.